src/utilities/browser-utils.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Allows us to use the documentMode for feature detection on globalThis.Document
+ */
+declare global {
+  interface Document {
+      documentMode?: any;
+  }
+}

```

Comment:
Simple newline for readability after line 8

Reply:
Fixed.